README.adoc

```diff
@@ -153,34 +153,34 @@ Implement the actual exception class and the mapper class to see how this mechan
 `src/main/java/io/openliberty/guides/inventory/client/UnknownUrlException.java`
```

Comment:
This block highlights the wrong file

Reply:
Opened a separate issue since README looks correct (also an issue on master).